spec/item_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative '../classes/item'
+
+describe Item do
+  let(:genre) { Genre.new('Fiction') }
+  let(:author) { 'J.K. Rowling' }
+  let(:label) { 'Harry Potter and the Philosopher\'s Stone' }
+  let(:publish_date) { Date.parse('1997-06-26') }
+
+  describe '#initialize' do
+    it 'creates a new item with specified genre, author, label, and publish date' do
+      item = Item.new(genre, author, label, publish_date)
+
+      expect(item.genre).to eq(genre)
+      expect(item.author).to eq(author)
+      expect(item.label).to eq(label)
+      expect(item.publish_date).to eq(publish_date)
+      # expect(item.archived).to eq(false)
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻